modules/apps/fragment/fragment-renderer-api/src/main/java/com/liferay/fragment/renderer/FragmentRenderer.java

```diff
@@ -42,6 +38,10 @@ public default String getKey() {
 
 	public String getLabel(Locale locale);
 
+	public default int getType() {
+		return FragmentConstants.TYPE_COMPONENT;
+	}
+
```

Comment:
Hey @brianchandotcom, with this rename, it seems to me as if the type refers to the FragmentRenderer. But this is not the renderer type but the fragmenty type.

One alternative would be to find another word other than "type" for the renderer. I'll look into it but will still resend you your pull as it is, to avoid blocking futher progress on this Story.